components/homme/test/reg_test/run_tests/test-list.cmake

```diff
@@ -44,10 +44,13 @@ SET(HOMME_TESTS
 
 IF (HOMME_ENABLE_COMPOSE)
   LIST(APPEND HOMME_TESTS
+    thetanh-moist-bubble-sl.cmake
```

Comment:
I had to rearrange some lists to support running C++/Kokkos tests in the optimized homme_integration test in addition to the BFB test.